recipe/bld.bat

```diff
@@ -1,29 +1,58 @@
-cd winbuild
+set CMAKE_CONFIG=Release
 
-if %ARCH% == 32 (
-    set ARCH_STRING=x86
-) else (
-    set ARCH_STRING=x64
-)
-
-REM This is implicitly using WinSSL.  See Makefile.vc for more info.
-nmake /f Makefile.vc MODE=dll VC=%VS_MAJOR:"=% WITH_DEVEL=%LIBRARY_PREFIX% ^
-         WITH_ZLIB=dll WITH_SSH2=dll DEBUG=no ENABLE_IDN=no ENABLE_SSPI=yes ^
-         MACHINE=%ARCH_STRING% ENABLE_UNICODE=yes
+:: Create and switch to CMake build directory
+mkdir build_%CMAKE_CONFIG%
+if errorlevel 1 exit 1
+pushd build_%CMAKE_CONFIG%
 if errorlevel 1 exit 1
 
-REM This is implicitly using WinSSL.  See Makefile.vc for more info.
-nmake /f Makefile.vc mode=static VC=%VS_MAJOR:"=% WITH_DEVEL=%LIBRARY_PREFIX% ^
-         WITH_ZLIB=dll WITH_SSH2=dll DEBUG=no ENABLE_IDN=no ENABLE_SSPI=yes ^
-         MACHINE=%ARCH_STRING% ENABLE_UNICODE=yes
+:: Build and install executable and dynamic libraries
+cmake -G "Ninja" ^
+    %CMAKE_ARGS% ^
+    -D CMAKE_BUILD_TYPE:STRING="%CMAKE_CONFIG%" ^
+    -D CMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -D CMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON ^
+    -D BUILD_CURL_EXE:BOOL=ON ^
+    -D BUILD_SHARED_LIBS:BOOL=ON ^
+    -D BUILD_STATIC_LIBS:BOOL=OFF ^
+    -D CURL_ZLIB:BOOL=ON ^
+    -D CURL_USE_LIBPSL:BOOL=OFF ^
+    -D CURL_USE_LIBSSH2:BOOL=ON ^
+    -D USE_WIN32_IDN:BOOL=OFF ^
+    -D CURL_USE_SCHANNEL:BOOL=ON ^
+    -D CURL_WINDOWS_SSPI:BOOL=ON ^
```

Comment:
Schannel implies SSPI. However without Schannel set SSPI is ignored. Idk if this behavior would change in the future. So just went ahead and set both

This seems to match what was done before